float64 height1
float64 height2
float64 height3
float64 height4
---
bool response